<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Transmission:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="transmission_edit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>40</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>+/-</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dtransmission_edit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="calculate_chk">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate transmission</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="direct_beam_chk">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>109</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Direct beam</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="beam_spreader_chk">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Beam spreader</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>671</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="widget_placeholder"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
